matchms/filtering/filters/add_compound_name.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from matchms.filtering.filters.base_spectrum_filter import BaseSpectrumFilter
+from matchms.typing import SpectrumType
+
+
+logger = logging.getLogger("matchms")
+
+class AddCompoundName(BaseSpectrumFilter):
```

Comment:
It might make more sense, to have the Classes in the same file that has the functional equivalent.

Reply:
So below the add_compound_name function

Reply:
Something that is hard to judge now is if the same functionality is still in the classes that was previously in the function, since we made quite some updates to matchms in the last few weeks. There is a risk that these changes are undone. I am not sure how we could best check this. Anyone any suggestions?